diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/IndentationRule.kt

```diff
@@ -209,15 +212,19 @@ class IndentationRule(configRules: List<RulesConfig>) : DiktatRule(
     }
 
     @Suppress("ForbiddenComment")
-    private fun visitWhiteSpace(astNode: ASTNode, context: IndentContext) {
-        context.maybeIncrement()
+    private fun IndentContext.visitWhiteSpace(astNode: ASTNode) {
+        require(astNode.isMultilineWhitespace()) {
+            "The node is $astNode while a multi-line $WHITE_SPACE expected"
```

Comment:
```suggestion
            "The node is ${astNode.elementType} while a multi-line $WHITE_SPACE expected"
```
I don't remember, does `ASTNode` have a decent `toString()`?

Reply:
Thanks for the heads-up.

`ASTNode` per se doesn't (it's an interface), but its abstract implementors (`TreeElement` and `FileElement`) do:

```java
    public @NonNls String toString() {
        return "Element(" + this.getElementType() + ")";
    }
```

I'd rather leave this as-is, because some implementations provide the `text` in addition to `IElementType`:

```java
    public String toString() {
        return "PsiParenthesizedExpression:" + this.getText();// 112
    }
```

Reply:
Ok, thanks for a detailed answer